### ISSUE TITLE
Search epi that its not on the IPS

### ISSUE THREAD

**Issue by joofio** [Gravitate-Health/mvp-issues#7]
`Being able to search by text on a repository of epi by the brand name or pharmaceutical product, we may have different (and more complicated levels to implement) like:
"1-manual identification - typing brand name or something
2-read code (GTIN, bidimensional)
3-picutre of the box
4-pill or tablet`

Focus on 1. then 2.
3 and 4 Out of Scope

**Comment by joofio** [comment]
@riccardo-leoni-dw  i believe this is just frontend right?

**Comment by RiccardoLeoni** [comment]
@joofio No, we'll need a service to perform the search and retrieve the medicines

**Comment by joofio** [comment]
I think we can do that with FHIR query alone. I will provide examples to check if that is enough

**Comment by joofio** [comment]
example:

search by name in productname of MP:
```GET https://fosps.gravitatehealth.eu/epi/api/fhir/MedicinalProductDefinition?name:contains=biktarvy```

Then get the composition 

```GET https://fosps.gravitatehealth.eu/epi/api/fhir/Composition?subject.identifier=EU/1/18/1289/001&_language=en&category=http://hl7.eu/fhir/ig/gravitate-health/CodeSystem/epicategory-cs|R```

If we want to search by code , we do:
```GET https://fosps.gravitatehealth.eu/epi/api/fhir/MedicinalProductDefinition?identifier=EU/1/18/1289/001```

 and then the same for composition

but for code, i believe we should talk about how to implement it. I can see it by taking photo of a QR code, code bar, GTIN or wtv but not someone writing the code by hand

What do you think?
